cmd/snap/cmd_debug_timings.go

```diff
@@ -177,7 +194,7 @@ func (x *cmdChangeTimings) Execute(args []string) error {
 	} else {
 		allEnsures = "false"
 	}
-	if err := x.client.DebugGet("change-timings", &timings, map[string]string{"change-id": chgid, "ensure": x.EnsureTag, "all": allEnsures}); err != nil {
+	if err := x.client.DebugGet("change-timings", &timings, map[string]string{"change-id": chgid, "ensure": x.EnsureTag, "all": allEnsures, "startup": x.StartupTag}); err != nil {
```

Comment:
could you use a struct? just so you can have `omitempty` all over the place i mean

Reply:
That's a good observation, however `DebugGet` is used in other places and I wanted to avoid changing it with this PR.